mobile/src/override/acradia-text.tsx

```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+import { TextProperties, StyleSheet, Text, ViewStyle, TextStyle } from 'react-native';
+
+const commonStyles = StyleSheet.create({
+    text: {
+        fontFamily: 'CenturyGothic'
+    }
+});
+
+export class ArcadiaText extends Component<TextProperties> {
```

Comment:
really????? is this a way to go???

Reply:
Yep, read the documentation. Spilled my smoothies.

Reply:
Check out `Text.defaultProps.style = { fontFamily: 'some-font' }` ?

Reply:
You all guys support using ArcadiaText component instead of defaultProps?

Reply:
It will be rewritten, if I pass some styles

Reply:
My code merges objects

Reply:
> You all guys support using ArcadiaText component instead of defaultProps?

I would prefer to merge styles than use this component )

Reply:
So, we've decided to call StyledText